scss/tech.scss

```diff
@@ -0,0 +1,69 @@
+$tech-height: 450px;
+$tech-width: 300px;
+$tech-logo-size: 100px;
+
+.tech-collection {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 20px 0;
```

Comment:
The margins left and right should be `auto`

Reply:
Also, try to add a max-width so it has the spaces on the sides that the design is showing. I'd say 1000px should be OK :smile: